third_party/2and3/click/exceptions.pyi

```diff
@@ -89,3 +89,8 @@ class FileError(ClickException):
 
 class Abort(RuntimeError):
     ...
+
+
+class Exit(RuntimeError):
+    def __init__(self, code: Optional[int] = ...) -> None:
```

Comment:
Thanks, one more thing: `code` should not be optional. Confusingly, `Optional` does not mean that an argument is optional (assigning a default values does that), it means that a value can also be `None`, which is not the case here.

Reply:
I removed this in 12e8628